File: server/src/main/java/com/fours/onlineschedulerapi/model/Joke.java
package com.fours.onlineschedulerapi.model;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import org.hibernate.annotations.CreationTimestamp;

import javax.persistence.*;
import java.util.Date;

@Entity
@Table(name = "jokes")
@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
public class Joke {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;
    private String title;
    private String text;
    private String labels;
    @Column(name = "created_at")
    @CreationTimestamp
    private Date createdAt;
    @ManyToOne
    private User user;

    public Joke(String title, String text, String labels){
        this.title = title;
        this.text = text;
        this.labels = labels;
    }
}

File: final project/humorme/src/main/kotlin/com/example/humorme/model/Joke.kt
package com.example.humorme.model

import java.util.Date
import javax.persistence.*

@Entity
data class Joke (
        @Id
        val id: String,
        val title: String,
        val text: String,
        val createdAt : Date
//        @ManyToOne(fetch = FetchType.LAZY)
//        val user: User
)